code/modules/crafting/recipes.dm

```diff
@@ -686,6 +686,16 @@
 	reqs = list(/obj/item/bedsheet = 1)
 	category = CAT_CLOTHING
 
+/datum/crafting_recipe/briefcase
+	name = "Hand made Briefcase"
+	result = /obj/item/storage/briefcase/crafted
+	time = 35
+	tools = list(TOOL_WIRECUTTER)
+	reqs = list(/obj/item/stack/sheet/cardboard = 1,
+				/obj/item/stack/sheet/cloth = 2,
```

Comment:
why cloth tho

Reply:
handle/padding. Also to prevent suitcase spamming for free-ish cargo points

Reply:
suitcases grant cargo points?